### DISCUSSION TITLE
Rendering a 3D Box Layer

### DISCUSSION THREAD

**Discussion by karllapuz** [visgl/deck.gl#7314] [Q&A] [answered]
Hi all, is there a way to render a cube as a layer where we can specify its central position, scale, and rotation (all in cartesian system)? It seems like the closest one to this is PolygonLayer, which involves calculating the vertices, or GridCellLayer, which doesn't look like has rotational property. I'm also looking into luma.gl if that's a possible route to achieve this. 

P.S. Coming from a Three.js background, would love to see how this all translates

Thanks!

**Comment by Pessimistress** [comment, marked as answer]
[SimpleMeshLayer](https://deck.gl/docs/api-reference/mesh-layers/simple-mesh-layer)

**Reply by karllapuz** [reply]
Ahh how'd I miss this! Thanks
